packages/amplify-environment-parameters/src/environment-parameter-manager.ts

```diff
@@ -85,6 +85,19 @@ class EnvironmentParameterManager implements IEnvironmentParameterManager {
     return !!this.resourceParamManagers[getResourceKey(category, resource)];
   }
 
+  async cloneEnvParamsToNewEnvParamManager(destEnvName: string): Promise<void> {
+    const destManager = (await ensureEnvParamManager(destEnvName)).instance;
```

Comment:
Can this be lifted into `cloneEnvParamManager`?

Reply:
No, it needs to be a method on the `EnvironmentParameterManager` class because it needs access to the private `resourceParamManagers` field.

Reply:
My comment was ambiguous, sorry. Not the whole function, just this line:

```ts
const destManager = (await ensureEnvParamManager(destEnvName)).instance;
```